.github/scripts/generate_ci_workflows.py

```diff
@@ -396,20 +392,6 @@ def generate_workflow_file(self, workflow_template: jinja2.Template) -> None:
             labels={LABEL_CIFLOW_DEFAULT, LABEL_CIFLOW_CPU, LABEL_CIFLOW_WIN}
         ),
     ),
-    CIWorkflow(
```

Comment:
We should also set the workflow below this one to be `LABEL_CIFLOW_DEFAULT`, right?

Reply:
ah good catch thanks